test/util/setup.js

```diff
@@ -0,0 +1,60 @@
+var loopback = require('loopback');
+
+var remoteConnector = require('../..');
+
+/**
+ * App helper
+ */
+function createRestAppAndListen(port) {
+  var app = loopback();
+
+  app.set('host', '127.0.0.1');
+  if (port) app.set('port', port);
+
+  app.use(loopback.rest());
+  app.locals.handler = app.listen();
+
+  return app;
+}
+module.exports.REST_APP = createRestAppAndListen;
```

Comment:
Why module.exports?  Just `exports.createRestAppAndListen`, same as all the other helper functions.

You can probably take all the functions and just export them deach direclty:

```
var helper = require('./utils/setup-helper');

helper.createRestAppAndListen();
helper.getMemoryDataSource();
helper.getRemoteDatasource();
...
```
Will make the code more descriptive in the other file too, instead of `setup.REST_APP` which doesn't look node-like at all.